xml/ay_bootloader.xml

```diff
@@ -174,46 +174,6 @@
 <screen>&lt;boot_root&gt;false&lt;/boot_root&gt;</screen>
     </listitem>
    </varlistentry>
-   <varlistentry>
-    <term>generic_mbr</term>
-    <listitem>
-     <para>
-      Write generic boot code to the MBR (will be ignored if
-      <literal>boot_mbr</literal> is set to <literal>true</literal>).
-     </para>
-<screen>&lt;generic_mbr config:type="boolean"&gt;false&lt;/generic_mbr&gt;</screen>
-    </listitem>
-   </varlistentry>
-   <varlistentry>
-    <term>gfxmode</term>
-    <listitem>
-     <para>
-      Graphical resolution of the &grub; screen (requires &lt;terminal&gt; to
-      be set to <literal>gfxterm</literal>).
-     </para>
-     <para>
-      Valid entries are <literal>auto</literal>, <literal><replaceable>
-      HORIZONTAL</replaceable>x<replaceable>VERTICAL</replaceable></literal>,
-      or <literal><replaceable>HORIZONTAL</replaceable>x<replaceable>VERTICAL
-      </replaceable>x<replaceable>COLOR DEPTH</replaceable></literal>. You can
-      see the screen resolutions supported by &grub; on a particular system by
-      using the <command>vbeinfo</command> command at the &grub; command line
-      in the running system.
-     </para>
-<screen>&lt;gfxmode&gt;1280x1024x24&lt;/gfxmode&gt;</screen>
-    </listitem>
-   </varlistentry>
-   <varlistentry>
-    <term>os_prober</term>
-    <listitem>
-     <para>
-      If set to <literal>true</literal>, automatically searches for operating
-      systems already installed and generates boot entries for them during the
-      installation.
-     </para>
-<screen>&lt;os_prober&gt;false&lt;/os_prober&gt;</screen>
-    </listitem>
-   </varlistentry>
```

Comment:
btw this is not removal, it is just making keys alphabetic sort. ( it is in first commit separated )

Reply:
:+1: